### DISCUSSION TITLE
Namespace error message is unclear

### DISCUSSION THREAD

**Discussion by msakrejda** [sidekiq/sidekiq#6038] [Q&A] [answered]
Hello,

We're trying to upgrade to Sidekiq 7. We're hitting [this error message](https://github.com/sidekiq/sidekiq/blob/1ba89bbb22d2fd574b11702d8b6ed63ae59e2256/lib/sidekiq/redis_client_adapter.rb#L66C1-L67):

> Your Redis configuration uses the namespace '...' but this feature isn't supported by redis-client. Either use the redis adapter or remove the namespace.

I've searched the docs and previous discussions, I don't understand what "use the redis adapater" means. The upgrade notes only mention that namespaces are no longer supported: https://github.com/sidekiq/sidekiq/blob/main/docs/7.0-Upgrade.md#redis-namespace

Is it still possible to use namespaces in 7? Either way, I'd love to see the error message improve (I'm happy to submit a patch, but at this point I don't know what's going on).

Thanks,
Maciek

**Comment by mperham** [comment, marked as answer]
You're right, it should just speak of removing the namespace. Namespaces are not supported in Sidekiq 7 so your own option is to remove them.
